shell/README.md

```diff
@@ -32,7 +32,16 @@ and then add the directory to your `$fpath` in your `.zshrc`, `.zsh_profile` or
 
 #### Oh my Zsh
 
-If you are using the popular [oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh) framework to manage your zsh plugins, move the file `exercism_completion.zsh` into `~/.oh-my-zsh/custom`.
+If you are using the popular [Oh My
+Zsh](https://github.com/robbyrussell/oh-my-zsh) framework to manage your zsh
+plugins, you need to move the file `exercism_completion.zsh` to a new custom
+plugin:
+
+    mkdir -p $ZSH_CUSTOM/plugins/exercism
+    cp ../shell/exercism_completion.zsh $ZSH_CUSTOM/plugins/exercism/_exercism
```

Comment:
Why does this path start with `..`?

Reply:
I included the `..` because the instructions for Bash and Zsh above have it., but I agree that it doesn't seem to have a clear purpose

Reply:
I removed the `../shell` in the latest commit, but left the Bash and Zsh sections alone.